include/iceflow/consumer-tlv.hpp

```diff
@@ -271,35 +272,35 @@ class ConsumerTlv {
         std::vector<std::vector<ndn::Block>> splitManifestBlocks;
 
         // grouping manifest data according to type
-        for (int i = 0; i < m_manifestDataTypes.size(); i++) {
+        for (int m_manifestDataType : m_manifestDataTypes) {
           std::vector<ndn::Block> tmp;
-          for (int j = 0; j < m_manifestBlocks[frame].size(); j++) {
-            if (m_manifestBlocks[frame][j].type() == m_manifestDataTypes[i]) {
-              tmp.push_back(m_manifestBlocks[frame][j]);
+          for (auto &j : m_manifestBlocks[frame]) {
+            if (j.type() == m_manifestDataType) {
```

Comment:
I would suggest using another variable name here to make it more clear what kind of data is being used here. Would `manifestBlock` be correct here?

```suggestion
          for (auto &manifestBlock : m_manifestBlocks[frame]) {
            if (manifestBlock.type() == m_manifestDataType) {
```

Reply:
```manifestBlock``` sounds fine here. I am correcting it.